bgpd/bgp_vty.c

```diff
@@ -2332,6 +2332,36 @@ DEFUN (no_bgp_timers,
 	return CMD_SUCCESS;
 }
 
+/* BGP minimum holdtime.  */
+
+DEFUN(bgp_minimum_holdtime, bgp_minimum_holdtime_cmd,
+      "minimum-holdtime (0-65535)",
```

Comment:
Is 0 a valid case?

Reply:
0 means that minimum-holdtime feature doesn't work.
This (=0) is default value.

Reply:
What is the difference between entering with value 0 versus "no bgp minimum.."?

Reply:
You are right. Value 0 equals "no bgp minimum..".
I would fix (1..65535).

Reply:
Fixed in b042667

Reply:
As long as hold timer range is 0 to 65535, the range allowed for bgp minimum-hold time command (this command) should also be 0 to 65535. This is how other vendors like Cisco supports. 

Hold timer of 0 has meaning where no keep alives are sent between the neighbors.